### PR TITLE
ci: adjust reviewers

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

File: .github/scripts/reviewers_add.js [hunk 1]
```diff
@@ -110,7 +106,6 @@ module.exports = async ({ github, context }) => {
   }
 
   if (labels.includes("vim-patch")) {
-    reviewers.add("seandewar");
```
Comment (clason) [inline .github/scripts/reviewers_add.js:113]: 🫡

File: .github/scripts/reviewers_add.js [hunk 2]
```diff
@@ -110,7 +102,6 @@ module.exports = async ({ github, context }) => {
...
     reviewers.add("zeertzjq");
```
Comment (dundargoc) [inline .github/scripts/reviewers_add.js:105]: TODO: remove

**Comment by dundargoc** [comment]
@neovim/core @neovim/triage lemme know if anyone wants to be removed/added from a label

File: .github/scripts/reviewers_add.js (hunk 1 above)
Comment (lewis6991) [reply .github/scripts/reviewers_add.js:113]: 🫡

File: .github/scripts/reviewers_add.js (hunk 2 above)
Comment (zeertzjq) [reply .github/scripts/reviewers_add.js:105]: ?

**Comment by gpanders** [comment]
You can remove me from the filetype label. Most of those are just vim-patch ports of new filetypes anyway. If there are ever questions about design/implementation of vim.filetype people can ping me if necessary (though it’s been in capable hands for quite some time now).

File: .github/scripts/reviewers_add.js (hunk 1 above)
Comment (seandewar) [reply .github/scripts/reviewers_add.js:113]: :saluting_face: 

Apologies for how long it's been! I expect to return eventually, but still not sure when...
Feel free to remove me as a reviewer in the meantime though. :hugs:

Comment (clason) [reply .github/scripts/reviewers_add.js:113]: You'll be welcome whenever 🤗  Take care, and drop by in chat sometime!

File: .github/scripts/reviewers_add.js (hunk 2 above)
Comment (dundargoc) [reply .github/scripts/reviewers_add.js:105]: Idk I assumed you didn't want it. I'll leave it then.

File: .github/scripts/reviewers_add.js [hunk 3]
```diff
@@ -105,12 +96,7 @@ module.exports = async ({ github, context }) => {
     reviewers.add("gpanders");
   }
 
-  if (labels.includes("typo")) {
-    reviewers.add("dundargoc");
-  }
```
Comment (dundargoc) [inline .github/scripts/reviewers_add.js:110]: No reason for this, people are merging typo PRs like they're speedrunning.

**Pull request merged**